src/tests/ftest/util/dfuse_utils.py

```diff
@@ -126,41 +127,67 @@ def remove_mount_point(self):
         """Remove dfuse directory
         Raises:
             CommandFailure: In case of error deleting directory
+
+        Try once with a simple rmdir which should succeed, if this
+        does not then try again with rm -rf, but still raise an error
         """
         # raise exception if mount point not specified
         if self.mount_dir.value is None:
             raise CommandFailure("Mount point not specified, "
                                  "check test yaml file")
 
-        dir_exists, _ = general_utils.check_file_exists(
+        dir_exists, clean_nodes = general_utils.check_file_exists(
             self.hosts, self.mount_dir.value, directory=True)
         if dir_exists:
+
+            target_nodes = self.hosts
+            self.log.debug('remove: hosts %s', self.hosts)
+            self.log.debug('remove: clean %s', clean_nodes)
+            target_nodes.remove(clean_nodes)
+            self.log.debug('remove: target %s', target_nodes)
+
+            cmd = "rmdir {}".format(self.mount_dir.value)
+            ret_code = general_utils.pcmd(target_nodes, cmd, timeout=30)
+            if len(ret_code) == 1 and 0 in ret_code:
+                return
+
+            failed_nodes = NodeSet(",".join(
+                [str(node_set) for code, node_set in ret_code.items()
+                 if code != 0]))
+
             cmd = "rm -rf {}".format(self.mount_dir.value)
-            ret_code = general_utils.pcmd(self.hosts, cmd, timeout=30)
-            if 0 not in ret_code:
+            ret_code = general_utils.pcmd(failed_nodes, cmd, timeout=30)
+            if len(ret_code) > 1 or 0 not in ret_code:
                 error_hosts = NodeSet(
                     ",".join(
                         [str(node_set) for code, node_set in ret_code.items()
                          if code != 0]))
                 raise CommandFailure(
-                    "Error removing the {} dfuse mount point on the following "
-                    "hosts: {}".format(self.mount_dir.value, error_hosts))
+                    "Error removing the {} dfuse mount point with rm on the "
+                    "following hosts: {}".format(self.mount_dir.value,
+                                                 error_hosts))
+            raise CommandFailure(
+                "Error removing the {} dfuse mount point with rmdir on the "
+                "following hosts: {}".format(self.mount_dir.value,
+                                             failed_nodes))
 
     def run(self):
         """ Run the dfuse command.
         Raises:
             CommandFailure: In case dfuse run command fails
         """
 
+        self.log.info('Starting dfuse at %s', self.mount_dir.value)
+
         # create dfuse dir if does not exist
         self.create_mount_point()
         # obtain env export string
-        env = self.get_default_env()
+        env = self.get_default_env(get_log_file(self.client_log))
```

Comment:
(pylint-undefined-variable) Undefined variable 'get_log_file'
(pylint-no-member) Instance of 'Dfuse' has no 'client_log' member